modules/le_log/le_log.h

```diff
@@ -0,0 +1,98 @@
+#ifndef GUARD_le_log_H
+#define GUARD_le_log_H
+
+#include <utility>
```

Comment:
That is a tricky one. It's easy enough to replace std::string with const char* but std::forward is extremely useful here

Reply:
since std::forward<T> is nothing but syntactic sugar for an unconditional cast to an r-value, we can substitute with: static_cast<T&&> - try and see if that works ;)